app/routes/meetings/components/MeetingDetail.css

```diff
@@ -21,3 +21,24 @@
   display: flex;
   margin-bottom: 10px;
 }
+
+/* The ghostly danger styled button looked ugly next to the filled green Accept button */
+.declineBtn {
+  background: var(--color-red-2);
+  border: none;
+  color: var(--color-white) !important;
+
+  &:hover:not(:disabled) {
+    background: var(--color-red-1);
+  }
+}
+
+/* stylelint-disable no-descending-specificity */
+html[data-theme='dark'] .declineBtn {
+  background: var(--color-red-1);
+  color: var(--color-black) !important;
+
+  &:hover:not(:disabled) {
+    background: var(--color-red-2) !important;
+  }
+}
```

Comment:
hmm, that's a lot of extra code for a small thing, but okey.

Reply:
Well, to me, it was worth it 🙃 It looked very off

Reply:
I removed this btw, and made it the "dark" button